package.json

```diff
@@ -11,12 +11,12 @@
   "scripts": {
     "clean": "rimraf dist lib",
     "build": "npm run clean && tsc",
-    "build-web": "webpack && webpack --env.production",
-    "build-node": "webpack --env.node && webpack --env.node --env.production",
     "_test": "jest",
     "test": "npm run lint && jest",
-    "lint": "tslint --project . --config tslint.json",
-    "lint:fix": "tslint --project . --config tslint.json --fix"
+    "lint": "eslint src",
+    "lint:fix": "npm run lint -- --fix",
+    "format": "prettier src --write && prettier test --write",
```

Comment:
I set this to just look at `src` and `test` and ignore HTML and JSON files. Do we want to run it on anything else?

Reply:
this is fine